plugins/auth-backend/src/providers/index.ts

```diff
@@ -14,4 +14,5 @@
  * limitations under the License.
  */
 
-export { createAuthProvider } from './factories';
+export { createAuthProvider, registerAuthProvider } from './factories';
+export * from './types';
```

Comment:
The index file is what decides what is being exported and should always list everything by name unless it's another index file, see https://backstage.io/docs/architecture-decisions/adrs-adr004

Reply:
Understood. I will try using these types to implement a `test` Provider to see what are the types I have to explicitly expose.